sensors/sensorhub-system-lane/src/main/java/com/botts/impl/system/lane/config/AxisCameraConfig.java

```diff
@@ -9,7 +9,22 @@
  */
 public class AxisCameraConfig extends FFMpegConfig{
 
-    @DisplayInfo(label = "Stream Path", desc = "Path for video stream. Only applies for the CUSTOM camera type.")
-    public String streamPath = "/axis-media/media.amp?adjustablelivestream=1&resolution=640x480&videocodec=h264&videokeyframeinterval=15";
+    public enum CodecEndpoint {
+        H264("/axis-media/media.amp?adjustablelivestream=1&resolution=640x480&videocodec=h264&videokeyframeinterval=15"),
+        MJPEG("/axis-media/media.amp?adjustablelivestream=1&resolution=640x480&videocodec=jpeg&videokeyframeinterval=15");
```

Comment:
Does the MJPEG have keyframes? We may want to remove that from the MJPEG endpoint.